fixes.inc

```diff
@@ -15723,6 +15923,57 @@ native BAD_GetServerVarAsBool(const varname[]) = GetServerVarAsBool;
 	#define GetVehicleSeats__( FIXES_GetVehicleSeats(
 #endif
 
+/**
+ * <remarks>
+ * Checks if component is rear bullbar.
+ * </remarks>
+ */
+
+#if FIX_API
+	stock bool:FIXES_IsComponentRearBullbar(componentid)
+	{
+		if(componentid == 1109 || componentid == 1110)
+		{
+			return true;
+		}
+		return false;
+	}
+
+	#if _FIXES_SAMP
+		#define _ALS_IsComponentRearBullbar
+		#define IsComponentRearBullbar( FIXES_IsComponentRearBullbar(
+	#endif
+
+	#define _ALS_IsComponentRearBullbar__
+	#define IsComponentRearBullbar__( FIXES_IsComponentRearBullbar(
+#endif
+
+/**
+ * <remarks>
+ * Checks if component is front bullbar.
+ * </remarks>
+ */
+
+#if FIX_API
+	stock bool:FIXES_IsComponentFrontBullbar(componentid)
```

Comment:
This is an interesting API extension, but it seems that if we're doing this one, we should do it for all the component slot types - all or nothing.  But since `GetVehicleComponentType` basically already does that, I don't think these functions are needed as API extensions.

Reply:
I used this to check if component is rear bullbar or front one. Within `GetVehicleComponentType`. I can make it static so it doesn't extend api.

Reply:
Well since this function is only called from one place in the code it isn't really needed at all.  Just inline the check.

Reply:
https://github.com/pawn-lang/sa-mp-fixes/pull/149/commits/8f85355c1cb216367e9b7b38783f0f31068372d3